Automattic/sensei
# Show student count/name on course selection modal

Part of https://github.com/Automattic/sensei/issues/4959

### Changes proposed in this Pull Request

In the course selection modal that's shown in the student management page for bulk action and context actions, we decided to show the student count in the description for the case of multiple students and the name of the student in the case of a single student. That is implemented here

### Testing instructions

- Create a few courses
- Create a few students
- Go to students management page
- Select actions from the student row context menu that opens the modal
- See if student's name is shown in the modal
- Check for add, remove and reset actions
- Click the checkbox for one or more student rows
- Click the bulk action course selection button from the top
- See if the name of the student is shown for the case of a single student, student count for multiple
- Please also check the texts for all 3 actions for both singular and plural cases if they are correct. That means 3 * 2 = 6 cases

### Screenshot / Video
Single student -
![image](https://user-images.githubusercontent.com/6820724/163994793-2363b499-bd99-4e78-8d50-dc9703392f54.png)

Multiple students -
![image](https://user-images.githubusercontent.com/6820724/163994659-076eb7b6-ce99-4a81-9219-3d0eac8859f4.png)
